components/sync_manager/src/manager.rs

```diff
@@ -249,7 +255,20 @@ impl SyncManager {
         } else {
             Some(&params.engines_to_change_state)
         };
-        let result = sync15::sync_multiple(
+        let settings = Settings {
+            fxa_device_id: params.fxa_device_id,
+            device_name: params.device_name,
+            device_type: if params.device_type == (DeviceType::Mobile as i32) {
```

Comment:
The list of device types is incomplete now, See https://searchfox.org/mozilla-mobile/source/application-services/components/fxa-client/src/http_client.rs#491-503

Reply:
I'll go ahead and mirror those here, since we now support fancy new [icons for them](https://searchfox.org/mozilla-central/rev/1d0e84d2ec2843924bc6e5ed6658f93439e12351/browser/base/content/browser-sync.js#882).